funcx_sdk/funcx/tests/unit/test_atomiccontroller.py

```diff
@@ -0,0 +1,84 @@
+import random
+
+from funcx.sdk.executor import AtomicController
+
+
+class _SS:
```

Comment:
I can see that `_SS` is used as a fake for the tests, and I "get it", but what does the name mean?

I also don't think it's necessary to name it with an underscore prefix since it's being declared in a test module. But that's 🤷

Reply:
Resolved per refactor for other comment.